sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -578,6 +579,25 @@ class EthService(
     }
   }
 
+  @RpcMethod("eth_getFeePayments")
+  def getFeePayments(blockId: String): AccountFeePaymentsInfo = {
+    if (blockId == null) {
+      return null
+    }
+
+    applyOnAccountView { nodeView =>
+      val feePayments =
+        nodeView.history
+          .getFeePaymentsInfo(blockId)
+
+      if (feePayments.isEmpty) {
+        throw new RpcException(RpcError.fromCode(RpcCode.ExecutionError, s"no fee payments found for block: $blockId"))
+      }
+
+      feePayments.get()
```

Comment:
Should it really be an exception if there are no fee payments? What about empty blocks? @alsala @i-Alex 

Anyways, this could be simplified to:
```scala
applyOnAccountView { nodeView =>
  nodeView.history
    .getFeePaymentsInfo(blockId)
    .getOrElse(throw new RpcException(
      RpcError.fromCode(RpcCode.ExecutionError, s"no fee payments found for block: $blockId")
    ))
}
```

Reply:
I wasn't able to use `getOrElse` method on Java's `Optional` class, so instead I used `ifPresent`. Now the method returns null if the payments weren't found in the specified block because that makes more sense to me also.

Reply:
current implementation is fine and coincide with another similar call: `eth_getFeePayments`